steam-client/src/main/java/com/avenga/steamclient/mapper/DotaMatchDetailsMapper.java

```diff
@@ -0,0 +1,181 @@
+package com.avenga.steamclient.mapper;
+
+import com.avenga.steamclient.model.steam.gamecoordinator.dota.match.*;
+import com.avenga.steamclient.protobufs.dota.DotaGCMessagesClient.CMsgGCMatchDetailsResponse;
+import com.avenga.steamclient.protobufs.dota.DotaGCMessagesCommon;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Map proto buff {@link CMsgGCMatchDetailsResponse} class to simple POJO {@link DotaMatchDetails} class.
+ */
+public class DotaMatchDetailsMapper {
+
+    public static DotaMatchDetails mapFromProto(CMsgGCMatchDetailsResponse.Builder builder) {
+        var match = builder.getMatch();
```

Comment:
reason for extracting this?

Reply:
We use it for mapping proto packet message to pojo in synchronous and asynchronous clients callback handlers, e.g. `MatchDetailsCallbackHandler` and `DotaMatchDetailsCallback`